lib/src/conflicts.rs

```diff
@@ -192,38 +192,53 @@ pub fn materialize_merge_result(
                     MergeHunk::Resolved(content) => {
                         output.write_all(&content)?;
                     }
-                    MergeHunk::Conflict(ConflictHunk {
```

Comment:
nit: in commit message:
> Now, since commit fe92db6166a7, when we materialize conflicts, we

The public commit is b84be06c0822.

Reply:
Thanks